upstream/upkubeless/upstream.go

```diff
@@ -26,29 +25,28 @@ import (
 
 const Namespace = "mattermost-kubeless-apps"
 
-type Upstream struct {
-	kubelessClient versioned.Interface
-}
+type Upstream struct{}
 
 var _ upstream.Upstream = (*Upstream)(nil)
 
 func MakeUpstream() (*Upstream, error) {
-	kubelessClient, err := kubelessutil.GetKubelessClientOutCluster()
+	_, err := kubelessutil.BuildOutOfClusterConfig()
```

Comment:
Are there docs that need to be updated?

Reply:
Yes, will do in the next PR (part 4, OpenFaas)